test/e2e/framework/kubernetes/install-retina-helm.go

```diff
@@ -58,6 +58,11 @@ func (i *InstallHelmChart) Run() error {
 	chart.Values["image"].(map[string]interface{})["pullPolicy"] = "Always"
 	chart.Values["operator"].(map[string]interface{})["tag"] = tag
 
+	// update helm chart to use images from acnrepo.azurecr.io when running e2e tests
+	chart.Values["image"].(map[string]interface{})["repository"] = "acnpublic.azurecr.io/microsoft/retina/retina-agent"
```

Comment:
Do we need to hardcode the `IMAGE_REGISTRY/IMAGE_NAMESPACE`?

Reply:
Fixed